Tasks/Apiatsionak/Task 3/task3.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime
+z, m, n,  = 'Без ',' ровно','Половина',
+h1 = {1: ' час', 2: ' два', 3: ' три', 4: ' четыре', 5: ' пять', 6: ' шесть', 7: ' семь', 8: ' восемь',
+9 :' девять', 10: ' десять', 11: ' одиннадцать', 12:' двенадцать', 13: ' час', 14: ' два', 15: ' три',
+16: ' четыре', 17: ' пять', 18: ' шесть', 19: ' семь', 20: ' восемь', 21: ' девять', 22: ' десять',
+23: ' одиннадцать', 24: ' двенадцать'}
+h2 = {1: ' первого', 2: ' второго', 3: ' третьего', 4: ' четвертого', 5: ' пятого', 6: ' шестого', 7: ' седьмого',
+8: ' восьмого', 9: ' девятого', 10: ' десятого', 11: ' одиннадцатого', 12: ' двенадцатого', 13: ' первого',
+14: ' второго', 15: ' третьего', 16: ' четвертого', 17: ' пятого', 18: ' шестого', 19: ' седьмого',
+20: ' восьмого', 21: ' девятого', 22: ' десятого', 23: ' одиннадцатого', 24: ' двенадцатого' }
+h3 = {1: 'Один час', 2: 'Два часа', 3: 'Три часа', 4: ' Четыре часа', 5: 'Пять часов',
+6: 'Шесть часов', 7: 'Семь часов', 8: 'Восемь часов', 9: 'Девять часов', 10: 'Десять часов',
+11: 'Одиннадцать часов', 12: 'Двенадцать часов', 13: 'Один час', 14: 'Два часа', 15: 'Три часа',
+16: 'Четыре часа', 17: 'Пять часов', 18: 'Шесть часов', 19: 'Семь часов', 20:'Восемь часов',
+21: 'Девять часов', 22: 'Десять часов', 23: 'Одиннадцать часов', 0: 'Ноль часов'}
+min1 = {1: 'Одна минута', 2: 'Две минуты', 3: 'Три минуты', 4: 'Четыре минуты', 5: 'Пять минут',
+ 6: 'Шесть минут', 7: 'Семь минут', 8: 'Восемь минут', 9: 'Девять минут', 10: 'Десять минут',
+ 11: 'Одиннадцать минут', 12: 'Двенадцать минут', 13: 'Тринадцать минут', 14: 'Четырнадцать минут',
+ 15: 'Пятнадцать минут', 16: 'Шестнадцать минут', 17: 'Семнадцать минут', 18: 'Восемнадцать минут',
+ 19: 'Девятнадцать минут', 20: 'Двадцать минут', 21: 'Двадцать одна минута', 22: 'Двадцать две минуты',
+ 23: 'Двадцать три минуты', 24: 'Двадцать четыре минуты', 25: 'Двадцать пять минут', 26: 'Двадцать шесть минут',
+ 27: 'Двадцать семь минут', 28: 'Двадцать восемь минут', 29: 'Двадцать девять минут', 31: 'Тридцать одна минута',
+ 32: 'Тридцать две минуты', 33: 'Тридцать три минуты', 34: 'Тридцать четыре минуты', 35: 'Тридцать пять минут',
+ 36: 'Тридцать шесть минут', 37: 'Тридцать семь минут', 38: 'Тридцать восемь минут', 39: 'Тридцать девять минут',
+ 40: 'Сорок минут', 41: 'Сорок одна минута', 42: 'Сорок две минуты', 43: 'Сорок три минуты', 44: 'Сорок четыре минуты'}
+#Словарь для превода чисел из цифр на буквенный русский
+min2 = {45: 'пятнадцати минут', 46: 'четырнадцати минут', 47: 'тринадцати минут', 48: 'двенадцати минут',
+49: 'одиннадцати минут', 50: 'десяти минут', 51: 'девяти минут', 52: 'восьми минут', 53: 'семи минут',
+54: 'шести минут', 55: 'пяти минут', 56: 'четырех минут', 57: 'трех минут', 58: 'двух минут', 59: 'одной минуты'}
```

Comment:
overcomplicated data structure with duplicates, -1 point for the unwise decision